home-data/src/main/java/com/softserveinc/ita/homeproject/homedata/entity/Invitation.java

```diff
@@ -0,0 +1,37 @@
+package com.softserveinc.ita.homeproject.homedata.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+
+@Getter
+@Setter
+@Builder
```

Comment:
We don't need a builder because of new mapper. MapStruct used it before but ModelMapper doesn't use it so delete it, we don't use builder in our entities since modelmapper were chosen.

Reply:
ok